lib/attributes_for/version.rb

```diff
@@ -1,3 +1,3 @@
 module AttributesFor
-  VERSION = "0.7.8"
+  VERSION = "0.7.9"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)